conformance/conf_tests/helpers.py

```diff
@@ -21,56 +21,114 @@
 from pygame.locals import SRCALPHA
 
 
-def create_surface():
+DEPTHS = (8, 16, 24, 32)
+
+
+class conformance_test_case(object):
+    """Tacks the appropriate attributes onto the test function"""
+
+    registry = {}
+
+    def __init__(self, test_file_name, depths=DEPTHS):
+        self.test_file_name = test_file_name
```

Comment:
If we're going to base the filename off the test name, is there really any value to dropping the test_ prefix? It does result in filenames like gen_32_test_something, but all the messages will refer to the full function name, which seems like a desirable feature.